### PR TITLE
Enable testing with table scan in MinMaxTest.cpp

### PR DESCRIPTION
**Pull request by mbasmanova**
Part of #6506

### CODE REVIEW

File: velox/functions/prestosql/aggregates/tests/MinMaxTest.cpp [hunk 1]
```diff
@@ -556,8 +552,7 @@ class MinMaxNTest : public functions::aggregate::test::AggregationTestBase {
         {},
         {"min(c0, 2)", "min(c0, 5)", "max(c0, 3)", "max(c0, 7)"},
         {expected},
-        /*config*/ {},
-        /*testWithTableScan*/ false);
+        /*config*/ {});
```
Comment (kagamiori) [inline velox/functions/prestosql/aggregates/tests/MinMaxTest.cpp:555]: Why do we still keep /\*config\*/ {}?

File: velox/functions/prestosql/aggregates/tests/MinMaxTest.cpp [hunk 2]
```diff
@@ -576,8 +571,7 @@ class MinMaxNTest : public functions::aggregate::test::AggregationTestBase {
...
```
Comment (kagamiori) [inline velox/functions/prestosql/aggregates/tests/MinMaxTest.cpp:574]: Ditto.

**Review by kagamiori — APPROVED** [review]
LGTM % two nits.

File: velox/functions/prestosql/aggregates/tests/MinMaxTest.cpp (hunk 2 above)
Comment (mbasmanova) [reply velox/functions/prestosql/aggregates/tests/MinMaxTest.cpp]: Not needed. Removed.

**Review by xiaoxmeng — APPROVED** [review]
@mbasmanova LGTM. Thanks!

**Pull request closed without merging**